x-pack/test/plugin_api_perf/plugins/task_manager_performance/index.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { initRoutes } from './init_routes';
+
+function avg(items) {
+  return (
+    items.reduce((sum, val) => {
+      return sum + val;
+    }, 0) / items.length
+  );
+}
+
+export default function TaskManagerPerformanceAPI(kibana) {
+  return new kibana.Plugin({
+    name: 'perfTask',
+    require: ['elasticsearch', 'task_manager'],
+
+    config(Joi) {
+      return Joi.object({
+        enabled: Joi.boolean().default(true),
+      }).default();
+    },
+
+    init(server) {
+      const taskManager = server.plugins.task_manager;
+      const performanceState = {
+        runningAverageTasks: 0,
+        averagesTaken: [],
+        runningAverageLeadTime: -1,
+        averagesTakenLeadTime: [],
+        leadTimeQueue: [],
+      };
+
+      setInterval(() => {
+        const tasks = performanceState.leadTimeQueue.length;
+        console.log(`I have processed ${tasks} tasks in the past 5s`);
```

Comment:
How did `console.log()` make it past the linter???  I guess I've never tried logging in a test, maybe the linter allows console access for tests?  TIL!

Reply:
I left that on purpose as it helps you visualise what's happening while it's running.
I figured it's ok in the test